doc/restapi/Roaster-REST-API.md

```diff
@@ -409,3 +476,276 @@ API.
 		}
 	}
 	```
+
+## Feed [/feed{?page}]
+### Get Global Feed [GET]
++ Relation: feed
++ Request: A global feed (application/json)
+	+ Parameters
+		+ page: 0 (number) - Page of global feed to return.
++ Response: 200
+	+ Body
+	```json
+	{
+		"items": [
+			{
+				"category": 0,
+				"username": "JustARegularUsername",
+				"title": "Woah, this user failed!",
+				"description": "Something happened [...]",
+				"time": "Mon Dec  3 10:51:52 CET 2018"
+			}, {
+				"category": 3,
+				"username": "JustAnotherRegularUsername",
+				"title": "Woah, this user also failed!",
+				"description": "Something happened [...]",
+				"time": "Mon Dec  3 09:48:12 CET 2018"
+			}
+		]
+	}
+	```
+	+ Schema
+	```json
+	{
+		"type": "object",
+		"properties": {
+			"items": {
+				"type": "array",
+				"items": {
+					"type": "object",
+					"properties": {
+						"category": {
```

Comment:
Just a thought, how do we handle avatars in the feed?

Reply:
I've forgotten to document the `/user/{username}/avatar` endpoint! Good catch.